Community-Supported/hyper-jupyter-kernel/hyper_kernel/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""A Hyper Jupyter kernel"""
+
+from .kernel import HyperKernel
+
+__version__ = '0.0'
```

Comment:
Shouldn't we put a version here?

Reply:
we could - but in the end it doesn't really make a difference, as we aren't really versioning this kernel anyway...

Reply:
What about putting 1.0?

Reply:
If we put 1.0, people could expect this to be an actual version number.
As we are not planning to actually use this version, I don't think we should put 1.0 in there.
Putting 0.0, we are at least making it explicit that we are not planning to use this version number

Reply:
OK :D

Reply:
turns out, everything is still working just fine, even after simply removing the version here